core/fuzz/Cargo.toml

```diff
@@ -0,0 +1,49 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+[package]
+name = "opendal-fuzz"
+version = "0.0.0"
+publish = false
+edition = "2018"
+
+[package.metadata]
+cargo-fuzz = true
+
+[dependencies]
+anyhow = "1.0.71"
+arbitrary = { version = "1.3.0", features = ["derive"] }
+libfuzzer-sys = "0.4"
+tokio = { version = "1", features = ["full"] }
+uuid = { version = "1.4.0", features = ["v4"] }
+
```

Comment:
Please use `taplo` to format toml files.

Reply:
solved by https://github.com/apache/incubator-opendal/pull/2609/commits/27d5bd0807ce9435396b86033a5961c56b9835b4